tests/operators/test_dbt_compile.py

```diff
@@ -84,20 +86,40 @@ def test_dbt_compile_non_existent_model(profiles_file, dbt_project_file, model_f
   NOW() AS field2
 """
 
-COMPILED_MODEL_4 = """
-with __dbt__cte__model_1 as (
-SELECT
-  123 AS field1,
-  'abc' AS field2
-)SELECT
-  field1 AS field1,
-  field2 AS field2,
-  SUM(CASE WHEN 'd' IN ('a', 'b', 'c') THEN 1 ELSE 0 END) AS field3
-FROM
-  __dbt__cte__model_1
-GROUP BY
-  field1, field2
-"""
+if DBT_INSTALLED_GTE_1_6:
+    COMPILED_MODEL_4 = dedent(
+        """
+        with __dbt__cte__model_1 as (
+        SELECT
+          123 AS field1,
+          'abc' AS field2
+        ) SELECT
+          field1 AS field1,
+          field2 AS field2,
+          SUM(CASE WHEN 'd' IN ('a', 'b', 'c') THEN 1 ELSE 0 END) AS field3
+        FROM
+          __dbt__cte__model_1
+        GROUP BY
+          field1, field2
+        """
+    )
+else:
+    COMPILED_MODEL_4 = dedent(
+        """
+        with __dbt__cte__model_1 as (
+        SELECT
+          123 AS field1,
+          'abc' AS field2
+        )SELECT
```

Comment:
nit:
```suggestion
        ) SELECT
```

Reply:
Or maybe not, looks like this is coming from before. Wondering if it breaks anything.